.github/release.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+changelog:
+  exclude:
+    labels:
+      - duplicate
+      - question
+      - invalid
+      - wontfix
+  categories:
```

Comment:
I'm guessing we can use what Github defaults to? Since we don't have the same label breakdown as Agones.

Or we just make it match what we have: https://github.com/googleforgames/global-multiplayer-demo/labels

Reply:
....uhmn, why are they all excluded? Don't we want some labels in our changelog?

_Very secure changelog, no changelog generated_ 😄